### ISSUE TITLE
The reproduced accuracy differs significantly from the reported accuracy in the paper

### ISSUE THREAD

**Issue by rp7sv** [RyanHTR/TBSI#11]
In another issue[https://github.com/RyanHTR/TBSI/issues/6](url), you mentioned that the SOT's pre-trained parameters were directly derived from OSTrack.
Therefore, I downloaded the pre-trained parameters of ViT-B from the link provided by the author of OSTrack.[https://drive.google.com/drive/folders/1ttafo0O5S9DXK2PX0YqPvPrQ-HWJjhSy](url)，However, the training results were far below the accuracy reported in the paper.
![微信图片_20240328174538](https://github.com/RyanHTR/TBSI/assets/79622142/2372fbec-86a7-49d1-8296-77ae723d5b2e)
I then downloaded the SOT pre-trained parameters provided by you[https://pan.baidu.com/s/1U42J6b3g1htma0OvmXRQCw?pwd=at5b#list/path=%2F](url)，and the accuracy significantly surpassed that of the OSTrack pre-trained parameters in just the first epoch.
![微信图片_20240328174839](https://github.com/RyanHTR/TBSI/assets/79622142/a1be3552-38ab-4006-80ec-95d57ef0d540)
I used Beyond Compare to perform a binary comparison of the weight files, and found that the pre-trained parameters you provided were different from those of OSTrack.
![微信图片_20240328180220](https://github.com/RyanHTR/TBSI/assets/79622142/878ac350-3be1-449b-8836-d52693b83b7b)
Both experiments were conducted in the same environment (2 A6000 GPUs with identical peripheral configurations).
I wonder what could be the reason behind these differences.

**Comment by RyanHTR** [comment]
We directly use the vitb_256_mae_ce_32x4_ep300/OSTrack_ep0300.pth.tar from OSTrack as the TBSITrack_SOT_Pretrained.pth.tar.
If you use OSTrack_ep0300.pth.tar as the cfg.MODEL.PRETRAIN_FILE, you may check the following line of code to make sure this checkpoint is completely loaded.
https://github.com/RyanHTR/TBSI/blob/6456fdd34b7bbd48f41c989f00d9172a81b395c9/lib/models/tbsi_track/tbsi_track.py#L134

**Comment by rp7sv** [comment]
This is my first time making subtle adjustments based on replicating an experiment, but I overlooked this detail. Thanks for your open-source code and timely clarification!!!
